### ISSUE TITLE
Redo all patch cables in the main network closet

### ISSUE THREAD

**Issue by mitar** [cloyne/network#92]
Followup to #87. We have replacement cables, we should redo all patch cables in the main network closet.

We have ethernet cables/patch cables:
* 2ft Black – for network closets around Cloyne, to connect switches to rooms, top row [done and used]
* 2ft Grey – for network closets around Cloyne, to connect switches to rooms, bottom row [done and used]
* 5ft Grey – to connect servers
* 5ft Yellow – for uplink in the server room
* 2ft Yellow – for uplink for switches in network closets around Cloyne [done and used]
* 2ft Green – to connect APs [done and used]
* 2ft Blue – for mesh antennas [done and used]

**Comment by mitar** [comment]
Not sure which color we wanted to use for patch cables in the main network closet for switches for rooms there. I think we might wanted to use black and grey, but we do not really have them. We would have to buy more.

**Comment by chloe-jones** [comment]
Two options: 
1) Keep our current Panduit version of cable management and buy the expensive keystone jacks that come with it: http://www.masterelectronics.com/panduit/cc5e88bl.html?p=12181809
2) Downgrade our cable management system to a less expensive system so we can use less expensive jacks in the future.

**Comment by mitar** [comment]
Hm, I think we can probably buy some other similar but cheaper ones?

BTW, around the house we have some cables we added to cabinets, which do not have such jacks, but are connected directly. If we buy jacks, we should probably connect them as well in this manner (cable -> jack, jack -> patch cable -> switch).

**Comment by jmrtns** [comment]
I agree with Mitar, I would much prefer to have regular keystone in the main network closet at-least, it may almost be worth it to have it in all the network closets/cabinets around the house. It's also not a downgrade, its an upgrade =) Keystone is much more universal and therefore cheaper, more versatile, and easier to find.

**Comment by mitar** [comment]
Also, that can probably come through BSCIT and it really makes it easier for future network managers to maintain things.

**Comment by jmrtns** [comment]
In other words, make keystone plates and jacks for all of our racks part of the BSCIT motion =p

**Comment by jmrtns** [comment]
You can also get some ideas here [](https://www.reddit.com/r/homelab/)

**Comment by mitar** [comment]
@hum42an: This has not been done? You should do it before leaving.
